tabs/led_strip.css

```diff
@@ -144,7 +144,7 @@
     margin: 3px 0;
 }
 
-.tab-led-strip button:hover {
+.tab-led-strip .not(colors) button:hover {
```

Comment:
Oh wow. Yes, it should have been. I've updated it so it doesn't need it and is black-text friendly.